### PR TITLE
fix(replay): Update text to reflect replay count

### LINKED ISSUE #74751: [Mobile Replay] Issue Detail Tab: Incorrect number of Users
The different users here are defined by the `user.id`, and there are 2 different users, but the app says 5:

Image

Example: https://sentry-sdks.sentry.io/issues/5607602092/replays/?project=4507596011798528&query=is%3Aunresolved+issue.priority%3A%5Bhigh%2C+medium%5D&referrer=issue-stream&selected_replay_index=0&statsPeriod=30d&stream_index=0

Relates to:
* https://github.com/getsentry/sentry/issues/74747

### PR DESCRIPTION
**Pull request by c298lee**
We are getting a replay count instead of a user count, so updating the text to reflect that.

Fixes https://github.com/getsentry/sentry/issues/74751

### CODE REVIEW

File: static/app/views/issueDetails/groupReplays/groupReplays.tsx [hunk 1]
```diff
@@ -270,9 +270,8 @@ function GroupReplaysTable({
       <ReplayCountHeader>
         <IconUser size="sm" />
         {t(
-          'Replay captured %s experiencing this issue across %s.',
-          tn('%s user', '%s users', replayCount ?? 0),
-          tn('%s event', '%s events', group.count)
```
Comment (michellewzhang) [inline static/app/views/issueDetails/groupReplays/groupReplays.tsx:275]: why did we get rid of this line?

**Review by michellewzhang — APPROVED** [review]

File: static/app/views/issueDetails/groupReplays/groupReplays.tsx (hunk 1 above)
Comment (c298lee) [reply static/app/views/issueDetails/groupReplays/groupReplays.tsx:275]: I wasn't sure how to word it properly with the events count, updated to bring it back, lmk if you have any suggestions

File: static/app/components/events/eventReplay/replayClipSection.tsx [hunk 2]
```diff
@@ -69,9 +69,10 @@ export function ReplayClipSection({event, group, replayId}: Props) {
     seeAllReplaysButton && replayCount && replayCount > 1 ? (
       <Fragment>
         <div>
-          {tct('Replay captured [replayCount] users experiencing this issue', {
-            replayCount,
-          })}
+          {t(
+            'There are %s with this issue.',
```
Comment (jas-kas) [inline static/app/components/events/eventReplay/replayClipSection.tsx:73]:
```suggestion
            'There are %s for this issue.',
```

File: static/app/views/issueDetails/groupReplays/groupReplays.tsx [hunk 3]
```diff
@@ -270,8 +270,8 @@ function GroupReplaysTable({
       <ReplayCountHeader>
         <IconUser size="sm" />
         {t(
-          'Replay captured %s experiencing this issue across %s.',
-          tn('%s user', '%s users', replayCount ?? 0),
+          'There are %s with this issue across %s.',
```
Comment (jas-kas) [inline static/app/views/issueDetails/groupReplays/groupReplays.tsx:273]:
```suggestion
          'There are %s for this issue across %s.',
```

**Review by jas-kas — APPROVED** [review]

File: static/app/views/issueDetails/groupReplays/groupReplays.tsx (hunk 1 above)
Comment (michellewzhang) [reply static/app/views/issueDetails/groupReplays/groupReplays.tsx:275]: with @jas-kas 's suggestion i think it sounds fine!

**Pull request merged**